pkg/manager/impl/execution_manager.go

```diff
@@ -650,6 +650,7 @@ func (m *ExecutionManager) launchSingleTaskExecution(
 		UserInputsURI:         userInputsURI,
 		SecurityContext:       executionConfig.SecurityContext,
 		LaunchEntity:          taskIdentifier.ResourceType,
+		Namespace:             namespace,
```

Comment:
Can we also make a backward compat change for older executions which dont have this namespace metadata set that it returns the project-domain which is widely used namespace

Reply:
I think we should respect the config if we're going to assume values rather than use the default mapping

Reply:
Yeah that works. wanted to make sure we have some data that we have valid data when we call the GET api even though the model wont have saved this when creating the execution